README.md

```diff
@@ -789,7 +789,7 @@ _See https://github.com/n1trux/awesome-sysadmin#distributed-filesystems_
 #### Command-line file upload
 
 - [Beauties](https://github.com/dsx/beauties) - Minimalist file sharing written in Go, to be used primarily from Unix shell (e.g. with curl). Can be built as a Debian package for easy install. `MIT` `Go`
-- [transfer.sh](https://transfer.sh) - Easy file sharing from the command line. ([Source Code](https://github.com/dutchcoders/transfer.sh)) `MIT` `Go`
+- [transfer.sh](https://github.com/dutchcoders/transfer.sh/) - Easy file sharing from the command line. ([Source Code](https://github.com/dutchcoders/transfer.sh)) `MIT` `Go`
```

Comment:
Remove `([Source Code](https://github.com/dutchcoders/transfer.sh))` as it is a duplicate link.

Reply:
Fair!

Reply:
? https://transfer.sh is official site (but https://transfersh.com is unofficial site, it's my instance). Why deleted?

Reply:
I don't get the point of this pull request. The  transfer.sh entry is fine as it is https://github.com/awesome-selfhosted/awesome-selfhosted#command-line-file-upload